pkgs/top-level/beam-packages.nix

```diff
@@ -50,14 +50,7 @@ in
       inherit wxSupport systemdSupport;
     };
 
-    erlang_27-rc3 = self.beamLib.callErlang ../development/interpreters/erlang/27-rc3.nix {
-      wxGTK = wxGTK32;
-      parallelBuild = true;
-      autoconf = buildPackages.autoconf269;
-      exdocSupport = true;
-      exdoc = self.packages.erlang_26.ex_doc;
-      inherit wxSupport systemdSupport;
-    };
+    erlang_27-rc3 = self.interpreters.erlang_27;
```

Comment:
Commit: `erlang: erlang_27-rc3 -> erlang_27 `

This change does not make sense.

This commit should be bumping erlang tag. And not redefining what `erlang_27-rc3` package is.

Reply:
I legit have no idea what you want here. I don't want to touch the `erlang` tag. At the end of this, I only want `erlang_27` to be initialized. This is why I am mega confused. I want to **only**

- add `erlang_27`
- remove `erlang_27-rc3`

Updating the erlang tag requires me to add all of these

```
      erlang = self.interpreters.${self.latestVersion};
      erlang_odbc = self.interpreters."${self.latestVersion}_odbc";
      erlang_javac = self.interpreters."${self.latestVersion}_javac";
      erlang_odbc_javac = self.interpreters."${self.latestVersion}_odbc_javac";
```

which I also **don't** want to do because I don't understand their existence.

Reply:
I understand _what they are_ but I don't understand _why they are here_

Reply:
Then, remove commit:

> erlang: erlang_27-rc3 -> erlang_27

Reply:
To bump erlang tag, you would have to only update `latestVersion` version. But yes, you don't have to do that.
But again, I only assumed that because you actually said without wanting to say that.
PR name has to be changed now, then.

Reply:
Done, if you are in the `#nix` channel of the Erlanger slack would love to discuss ways we could simplify the beam tags. Looks like you did the replacement of `erlangR` -> `erlang_` but we never deprecated the old tags. Also looks like there are some dangling aliases for old versions too.

Reply:
I also started a 🧵 there on updating the latest version to gather feedback.

Reply:
Sorry for being a pain in your butt 😆

Reply:
> if you are in the `#nix` channel of the Erlanger slack 

Sorry. I don't use slack and I don't know this channel. I stopped with Erlang/Elixir several years ago. I still appreciate the technology. But I'm no longer active in it.

> would love to discuss ways we could simplify the erlang tags.

What do you mean?

> Looks like you did the replacement of `erlangR` -> `erlang_` but we never deprecated the old tags. 

Feel free to do it. It's been a while.

> Also looks like there are some dangling aliases for old versions too.

Erlang & Elixir are lacking upkeep. There are few people doing upkeep. The work you are doing here is important. Because you took an interest in it. I remember finding several problems in how things are structure for Elixir. Like, overrides not working, or some other issue that I don't recall correctly. It is possible to use a better Nix structure, but I suggest you to start slowly with basic things before diving deeper.

I remember having thought about a way to speed up builds with cache for dependencies. But it never materialized. Nix and Elixir is a good match. I wish I had found Nix earlier when I was still using Erlang/Elixir.

Communication today was messy. But I think, you came out of it with a better understanding.

Reply:
> Sorry. I don't use slack and I don't know this channel. I stopped with Erlang/Elixir several years ago. I still appreciate the technology. But I'm no longer active in it.

No worries.

> would love to discuss ways we could simplify the erlang tags.
> > What do you mean?

For example, are the `_odbc`, `_javac`, and `_odbc_javac` widely used? They aren't exposed in `all-packages.nix` so are they cached? AFAICT, they aren't very discoverable and _could_ be nice to trim.

> Looks like you did the replacement of erlangR -> erlang_ but we never deprecated the old tags.
Feel free to do it. It's been a while.

Awesome, I'll add it to my list.

> I remember finding several problems in how things are structure for Elixir. Like, overrides not working, or some other issue that I don't recall correctly. It is possible to use a better Nix structure, but I suggest you to start slowly with basic things before diving deeper.

Thanks for the extra context!

> Communication today was messy. But I think, you came out of it with a better understanding.

Yeah, partially my fault. I should have explained better in the description **what** I wanted to accomplish. I most certainly did learn a bunch.

I am also glad we **didn't** end up doing the `_odbc_javac` versions because we can see who is relying on them.